scripts/init-auth.py

```diff
@@ -22,4 +22,5 @@
     ClientId=sys.argv[1]
 )
 
-print(json.dumps(response['AuthenticationResult'], indent=2))
+id_token = response['AuthenticationResult']['IdToken']
+print(id_token)
```

Comment:
Apply the same change to `provision-user.py`?

Reply:
Done. Also updated GH workflow and install scripts per your other comment